README.md

```diff
@@ -1,2 +1,2 @@
-# hello-world
-Getting started with hello-world
+# Hello World!
```

Comment:
Now the maintainer reviews the lines changed and might have additional comments or complaints.
The contributors would argue their point here about a particular change.